ffb-common/src/main/java/com/fumbbl/ffb/model/RosterSkeleton.java

```diff
@@ -56,6 +57,9 @@ public IXmlSerializable startXmlElement(Game game, String pXmlTag, Attributes pX
 			if (StringTool.isProvided(pXmlAttributes.getValue(_XML_ATTRIBUTE_ID))) {
 				fId = pXmlAttributes.getValue(_XML_ATTRIBUTE_ID).trim();
 			}
+			if (StringTool.isProvided(pXmlAttributes.getValue(_XML_ATTRIBUTE_TEAM))) {
```

Comment:
This logic mirrors the one found in `Roster.java`: https://github.com/christerk/ffb/blob/5e6dc46c6c40f3e39b16c0a8b14dfa5aee9781ac/ffb-common/src/main/java/com/fumbbl/ffb/model/Roster.java#L250

Reply:
With this you are always setting the fId field to the team value if a team field is present, even if we have an id already. 

Additionally this parsing is not only used in standalone mode but a also in regular production mode. We should not change this behaviour or we will encounter nasty side effects sooner or later.

Generally with this approach you would have to save the same roster multiple times in case you e.g. want to test 2 different dark elf teams. Maybe there is no other way but why not have RosterCache populate a second map with team ids mapped to roster files?